test/switchboard/default-switchboards/FastSwitchboard.t.sol

```diff
@@ -5,16 +5,16 @@ import "../../Setup.t.sol";
 
 contract FastSwitchboardTest is Setup {
     bool isFast = true;
-    uint256 immutable remoteChainSlug = uint32(uint256(2));
-    bytes32 immutable packetId = bytes32(0);
+    uint32 immutable remoteChainSlug = uint32(2);
+    bytes32 immutable packetId = bytes32(uint256(2) << 224);
```

Comment:
use the c++ pattern like other tests instead of hard coding 1, 2.
this avoids human error where 2 constants end up having same value.